ecs-image-build/docker_start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
this is just ../docker_start.sh but without the npm i 
could they be combined into one?
if [ $1 = "install" ]; then
  npm i
fi

Reply:
This one in used for the ECS image whereas ../docker_Start.sh is used for the local environment. Keeping them separate is the standard being applied to all migrated services.